ui/src/pages/Alert/Shield/ShieldDrawerForm.tsx

```diff
@@ -71,9 +72,11 @@ export default function ShieldDrawerForm({
     } else {
       form.setFieldValue('endsAt', dayjs(time));
```

Comment:
I found that there are dayjs and moment in the `package.json`. It would be a good choice to use only one of them.

Reply:
You are right, `dayjs` should be able to completely replace `moment`.

Reply:
`dayjs` seems to be more light-weight and easy to use

Reply:
And `antd` recommends using `dayjs`.